### PR TITLE
Config fixes

### PR DESCRIPTION
**Pull request by Spomky**
This should fix several issues with the icons, screenshots...

### CODE REVIEW

File: config/packages/pwa.yaml [hunk 1]
```diff
@@ -12,14 +12,14 @@ pwa:
         scope: "/"
         display_override: ['fullscreen', 'minimal-ui', 'window-controls-overlay']
         id: "/"
-        start_url: "./"
+        start_url: "/"
         icons:
             # https://www.veryicon.com/icons/miscellaneous/smart-site/noise-3.html
             - src: "images/logo.png"
-              sizes: "any"
```
Comment (Spomky) [inline config/packages/pwa.yaml:19]: `any` is reserved for vector images (SVG)

File: config/packages/pwa.yaml [hunk 2]
```diff
@@ -12,14 +12,14 @@ pwa:
...
+              sizes: [32, 48, 72, 96, 120, 128, 144, 180, 192, 256, 384, 512]
 
         screenshots:
-            - src: /screenshots/screenshot-1200x1100.png
```
Comment (Spomky) [inline config/packages/pwa.yaml:22]: If you want to refer to the asset, use the asset notation.
When the path starts with a `/`, it is understood as a relative path from the public root.

File: config/packages/pwa.yaml [hunk 3]
```diff
@@ -33,15 +33,16 @@ pwa:
...
-                - {path: "app_planet_show", params: {id: 2}}
-                - {path: "app_planet_show", params: {id: 3}}
-                - {path: "app_planet_show", params: {id: 4}}
-                - {path: "app_planet_show", params: {id: 5}}
-                - 'app_homepage'
-                - 'app_voyage_index'
-                - 'app_planet_index'
-#                - 'app_about'
-            page_fallback: 'app_homepage'
-            static_regex: '/\.(css|js|json|xml|txt|mp3|map|webmanifest)$/'
```
Comment (Spomky) [inline config/packages/pwa.yaml:47]: You should let the default behavior. It also includes images served by asset mapper.

File: config/packages/pwa.yaml [hunk 4]
```diff
@@ -33,15 +33,16 @@ pwa:
         src: "sw.js"
         skip_waiting: true
         workbox:
-            warm_cache_urls:
```
Comment (Spomky) [inline config/packages/pwa.yaml:36]: The cache sections changed in 1.1.x.
The previous configuratin is valid, but difficult to read.

File: config/packages/pwa.yaml (hunk 3 above)
Comment (tacman) [reply config/packages/pwa.yaml:47]: I remember adding that when we were discussing caching mp3s.  Adding pages to the page cache from a service feels like the next most important thing now.  I think there's an issue open already.

**Comment by tacman** [comment]
Thanks for these fixes!

**Pull request merged**